pkg/etcdenvvar/etcd_env.go

```diff
@@ -52,9 +52,12 @@ func getEtcdEnvVars(envVarContext envVarContext) (map[string]string, error) {
 	// TODO once we are past bootstrapping, this restriction shouldn't be needed anymore.
 	//   we have it because the env vars were not getting set in the pod and the static pod operator started
 	//   rolling out to another node, which caused a failure.
-	if len(envVarContext.status.NodeStatuses) < 3 {
-		return nil, fmt.Errorf("at least three nodes are required to have a valid configuration")
-	}
+	// TODO: figure out how risky is commenting the lines below
+	//  We are able to scale from 3->4->5, which means we would be abel to scale
+	//  from bootstrap node the same way.
+	//if len(envVarContext.status.NodeStatuses) < 3 {
```

Comment:
FTR: removing this is very risky and can cause failure as indicated by comments above